app/javascript/react/components/Board.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import Square from "./Square"
+
+const Board = (props) => {
+  const [selectedSquare, setSelectedSquare] = useState(null)
+
+  let boardData = []
+  let letters = ["h", "g", "f", "e", "d", "c", "b", "a"]
+  for (let i = 1; i <= 8; i++) {
+    letters.forEach((letter) => {
+      boardData.unshift(
+        <Square
+          row={i}
+          column={letter}
+          selectedSquare={selectedSquare}
+          setSelectedSquare={setSelectedSquare}
+        />
+      )
+      if (letter === "a") {
+        boardData.unshift(
+          <br/>
```

Comment:
I wouldnt add a `br` here. I see why you did for the sake of maybe styles, but I can see this running into issues later on. use CSS styling inside, or style the `a` square element with a unique id that adds the spacing you want